### PR TITLE
Adds assignee list override

### LINKED ISSUE #36: Enable specification of a separate assignees list
**Is your feature request related to a problem? Please describe.**

Some of the reviews are frustrated when they are assigned but they're not responsible for the PR. They're just reviewers.

**Describe the solution you'd like**

I'd like to be able to have a separate `assignees` list.

**Describe alternatives you've considered**

I can remove the unwanted assignees from the reviewers list, but then they'd not get reviews, either.

### PR DESCRIPTION
**Pull request by colindean**
Fixes #36

Note that I took a quick stab at this having never worked in Typescript previously. Running `npm run build` and `npm start` seemed to work, so I'm hoping that the build system caught any errors I may have introduced because I don't have the bandwidth to set everything up to test it.

Typescript is pretty neat, especially when using vim-polyglot and I got syntax highlighting and linting in the editor without having to configure a thing!

### CODE REVIEW

File: src/handler.ts [hunk 1]
```diff
@@ -43,7 +45,12 @@ export async function handlePullRequest (context: Context): Promise<void> {
   }
 
   if (config.addAssignees) {
-    result = await pullRequest.addAssignees(owner, repo, prNumber, reviewers)
+    const assignees: string[] = config.assignees ?
+      chooseUsers(owner, config.assignees, config.numberOfAssignees || config.numberOfReviewers)
```
Comment (colindean) [inline src/handler.ts]: I'm not certain on the `||` usage here because I'm not certain how the `?` affects things in the interface AppConfig…

**Comment by kentaro-m** [comment]
@colindean 
Thanks for your feature request and PR.

First of all, I share what I am concerned about. I'd like to accept your changes as much as possible, but since this application is already being used by some users, I would like to be compatibility.

Currently, I am checking compatibility. If you are in a hurry, try application self-hosting.

c.f. Running the application with self-hosting
https://github.com/kentaro-m/auto-assign#glitch

**Comment by colindean** [comment]
Thanks for taking a look. I'm not in a hurry at the moment.

**Comment by colindean** [comment]
Would it faciliate merging if I instead put the use of the attendees list behind a flag, defaulting to false, where true uses the new behavior and false falls back to the old behavior of using the reviewers list?

**Comment by kentaro-m** [comment]
@colindean 
I have confirmed your changes, so I merged. Thanks.
By the way, I did not notice that the test failed, because the CircleCI build was not working by my mistake. I would like to create a PR to add tests on your changes.

**Pull request merged**